frontend/src/pages/modelServing/screens/metrics/BiasConfigurationTable.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { Button, ToolbarItem } from '@patternfly/react-core';
+import Table from '~/components/table/Table';
+import DashboardSearchField, { SearchType } from '~/concepts/dashboard/DashboardSearchField';
+import BiasConfigurationTableRow from '~/pages/modelServing/screens/metrics/BiasConfigurationTableRow';
+import { columns } from './tableData';
+import { MockBiasConfigurationType } from './mockConfigurations';
+
+type BiasConfigurationTableProps = {
+  configurations: MockBiasConfigurationType[];
+};
+
+const BiasConfigurationTable: React.FC<BiasConfigurationTableProps> = ({ configurations }) => {
+  const [searchType, setSearchType] = React.useState<SearchType>(SearchType.NAME);
+  const [search, setSearch] = React.useState('');
+  const filteredConfigurations = configurations.filter((configuration) => {
+    if (!search) {
+      return true;
+    }
+
+    // TODO: add more search types
+    switch (searchType) {
+      case SearchType.NAME:
```

Comment:
Ahh OK should have checked the code before my comment - yeah it makes sense to leave it until you have proper data :)